pkg/ibmvpc/parser.go

```diff
@@ -1430,6 +1430,9 @@ func getLoadBalancerIPs(vpcConfig *vpcmodel.VPCConfig,
 			return nil, err
 		}
 		// we want a pip for every subnet:
+		if len(subnetsBlocks.subnetBlocks(*subnetObj.CRN)) > 1{
+			logging.Warnf("subnet %s is spit by filters, more than one private IP is created\n", *subnetObj.Name)
```

Comment:
- please use `logging.Debugf`
- please add details: what private IPs are chosen for this subnet

Reply:
done